pkg/config/config_template.yaml

```diff
@@ -1738,6 +1738,17 @@ api_key:
 #
 # dogstatsd_stats_enable: false
 
+## @param dogstatsd_stats_logging - boolean - optional - default: true
+## Set to true to write DogstatsD metrics received by the Agent to dogstats_stats log files.
+#
+# dogstatsd_stats_logging: true
```

Comment:
Should this be `dogstatsd_logging_enabled` or something similar?

Reply:
👍